### PR TITLE
product list

### PR DESCRIPTION
**Pull request by anupkgurung**
productlist

### CODE REVIEW

**Comment by aurobindoGupta** [comment]
nice work!!
sidenav looks a little cluttered preferably look into its spacing.

**Review by aurobindoGupta — APPROVED** [review]

**Review by aniketsingh13 — APPROVED** [review]
look good to me

File: style/homepage.css [hunk 1]
```diff
@@ -61,6 +61,36 @@
...
+    justify-content: space-between;
+}
+.product-card-footer .btn{
+    border-radius: 0;
+}
+.product-card-footer .btn-secondary-outline {
+    border-bottom-left-radius: 5px;
+    border-bottom-right-radius: 5px;
+}
+.cateogory li input,
```
Comment (rajkishancode) [inline style/homepage.css]: There's a typo ,you can correct this.

**Review by rajkishancode — APPROVED** [review]
Rest looks good .

File: page/productlist.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://metaui.netlify.app/css/main.css">
+    <link rel="stylesheet" href="../style/commons.css">
+    <link rel="stylesheet" href="../style/header.css">
+    <link rel="stylesheet" href="../style/homepage.css">
+    <title>Product List</title>
+</head>
+<body>
+    <div>
+        <div id="" class="header-bar">
```
Comment (akshay-rajput) [inline page/productlist.html]: remove id="" if not using it.

File: page/productlist.html [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+            <div class="topbar">
+                <div><img class="logo" src="../image/logo.jpg" alt=""></div>
+                <div class="search-bar-head">
+                    <input type="text" class="search input-width" placeholder="Search">
+                </div>
+                <div class="">
+                    <ul class="list-inline">
```
Comment (akshay-rajput) [inline page/productlist.html]: remove list-inline from ul

File: page/productlist.html [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+                        <li><a class="link-style-none" tabindex="1">Books</a></li>
+                        <li><a class="link-style-none" tabindex="1">New Arrivals</a></li>
+                        <li><a class="link-style-none" tabindex="1">Best Seller</a></li>
+                        <li><a class="link-style-none" tabindex="1">Pre-Order</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+        
+        <div id="main" class="content-body">
```
Comment (akshay-rajput) [inline page/productlist.html]: use main tag here

File: page/productlist.html [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+                        </ul>
+                    </div>
+                    <div>
+                    </div>
+                </aside>
+            </div>
+            <div id="content">
+                <main class="content-container">
+                    <div class="item">
+                        <div class='card-container card-container--verticle product-card-container'>
```
Comment (akshay-rajput) [inline page/productlist.html:99]: keep your css class name convention consistent

File: page/productlist.html [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+                                <button class="btn btn-secondary-outline w-100 ">Remove from Wishlist</button>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="item">
+                        <div class='card-container card-container--verticle product-card-container'>
+                            <span class="badge text-badge z-1 m-0">20%</span>                         
+                            <div class="card-header">                                
+                                <div class="card-img-container position-rel">
+                                    <img class="responsive img-radius position-abs" src="../image/book/digitphototwo.jpg" alt="The Digital Photography vol-II">
```
Comment (akshay-rajput) [inline page/productlist.html:160]: if possible use webp images. look up the support for webp though

File: style/homepage.css [hunk 7]
```diff
@@ -61,6 +61,36 @@
     color: var(--royalblue-bg);
     font-weight: 200;
 }
+.aside {
+    position: fixed;
+}
+.product-card-container {
+    height: 18.7rem;
```
Comment (akshay-rajput) [inline style/homepage.css]: giving fix height or width might create issues with responsiveness. try to find a better way to achieve what you want

**Review by akshay-rajput — CHANGES_REQUESTED** [review]
make these changes and ping me.

**Pull request merged**